### PR TITLE
check tls config before handshake

### PR DESCRIPTION
**Pull request by Lagovas**
now it's panic with nil dereference when you didn't pass tls path to key/cert/ca cert and when you initialize tls connection with db
this PR log error message about it and tell that you need to provide such files if you want to run acraserver with tls support app<->db

### CODE REVIEW

File: decryptor/mysql/response_proxy.go [hunk 1]
```diff
@@ -174,6 +175,20 @@ func (handler *MysqlHandler) ClientToDbProxy(errCh chan<- error) {
 			firstPacket = false
 			handler.clientProtocol41 = packet.ClientSupportProtocol41()
 			if packet.IsSSLRequest() {
+				if handler.tlsConfig == nil {
+					log.Errorln("To support TLS connections you must pass TLS key and certificate for AcraServer that will be used" +
```
Comment (shadinua) [inline decryptor/mysql/response_proxy.go]: Looks like the space was missed at the end of this line (and below in the same block).

**Comment by vixentael** [comment]
Is there any easy way how we can test it?

File: tests/test.py [hunk 2]
```diff
@@ -1548,8 +1548,7 @@ def test_only_alpha_client_id(self):
         # call with directory separator in key name
         self.assertEqual(create_client_keypair(POISON_KEY_PATH), 1)
 
-
-class SSLPostgresqlConnectionTest(AcraCatchLogsMixin, HexFormatTest):
+class SSLPostgresqlMixin(AcraCatchLogsMixin):
     ACRA2_PORT = BaseTestCase.ACRA_PORT+1000
```
Comment (storojs72) [inline tests/test.py]: Why did you decide to plus 1000? Why not 2000 or 10000?

Comment (Lagovas) [reply tests/test.py]: chosen manually. we ran tests on ~5-10 go versions and for each we use ACRA_PORT+1. Thought that 100 may have intersection with any PROXY_PORT

Comment (storojs72) [reply tests/test.py]: There is no probability that port will be greater than max value - 65535?

Comment (Lagovas) [reply tests/test.py]: No, because it may be overrided with TEST_ACRA_PORT env variable that in tests use 10003 port as default

**Pull request merged**